api/v1alpha2/zz_generated.conversion.go

```diff
@@ -508,6 +508,7 @@ func autoConvert_v1alpha2_ClusterSpec_To_v1alpha3_ClusterSpec(in *ClusterSpec, o
 }
 
 func autoConvert_v1alpha3_ClusterSpec_To_v1alpha2_ClusterSpec(in *v1alpha3.ClusterSpec, out *ClusterSpec, s conversion.Scope) error {
+	// WARNING: in.Paused requires manual conversion: does not exist in peer-type
```

Comment:
Are you going to do the annotation magic for conversion?

Reply:
Yeah, I'll put a PR out later today